packages/lodestar/src/tasks/index.ts

```diff
@@ -13,6 +14,12 @@ import {StatesArchiver} from "./tasks/archiveStates";
 import {IBeaconSync} from "../sync";
 import {InteropSubnetsJoiningTask} from "./tasks/interopSubnetsJoiningTask";
 import {INetwork, NetworkEvent} from "../network";
+import {JobQueue} from "../util/queue";
+
+/**
+ * Minimum number of epochs between archived states
+ */
+export const MIN_EPOCHS_PER_DB_STATE = 1024;
```

Comment:
this is the same value and concept as `PERSIST_STATE_EVERY_EPOCHS` no?  why not just use that const instead?

Reply:
I think these lines can be deleted?